charts/openebs-rawfile-localpv/templates/node-plugin.yaml

```diff
@@ -86,10 +86,10 @@ spec:
             - name: socket-dir
               mountPath: /csi
             - name: mountpoint-dir
-              mountPath: /var/lib/kubelet
+              mountPath: {{ .Values.node.k8s_mountpoint | default "/var/lib/kubelet"  }}
```

Comment:
pvs are not mounted if this value is not matching the path associated with the volume with the same name `mountpoint-dir` with value `var/lib/k0s/kubelet`